src/controllers/chats.ts

```diff
@@ -205,10 +229,10 @@ export async function createGroupChat(req: Req, res) {
     }
   })
 
-  let chatParams: any = null
```

Comment:
Is it ok to remove the null here?

Reply:
i try to avoid null as much as possible and replace it with undefined. i would set the type to `Chat | undefined` if it could be undefined, but that's not the case here. `chatParams` in never used where it could be undefined: after the if block you know it is not undefined because it is assigned a value in both possible paths (and before the if it is not used)

Reply:
ya so probably fine just to do `let chatParam;`?